### PR TITLE
Initial project structure

### LINKED ISSUE #1: Initial Project

### PR DESCRIPTION
**Pull request by bruxita2312**
Fixes: #1 
- gradle structure
- Sign In is ready to review

### CODE REVIEW

File: src/main/java/com/jalasoft/sdfc/pages/Signin.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    public Signin() {
+        driver.get("https://login.salesforce.com/");
+    }
+
+    public Header loginAs(String userName, String password) {
+        action.setValue(userNameTextField, userName);
+        action.setValue(passwordTextField, password);
+        action.click(loginButton);
+
+        wait.until( ExpectedConditions.titleIs("Verify Your Identity | Salesforce"));
```
Comment (acamacho82) [inline src/main/java/com/jalasoft/sdfc/pages/Signin.java]: I think this is not necessary anymore, remove it if this is the case.

File: src/test/resources/features/Signin.feature [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Feature: Signin
+
+  Scenario: Salesforce sign in
+    Given I sign in as "rpfh1@mailinator.com" user and "654321*Aa"
```
Comment (acamacho82) [inline src/test/resources/features/Signin.feature]: replace the user credentials by **username** and "password"

**Review by acamacho82 — CHANGES_REQUESTED** [review]
I have some comments, please see below

File: src/main/java/com/jalasoft/sdfc/pages/Signin.java (hunk 1 above)
Comment (edgarquispe) [reply src/main/java/com/jalasoft/sdfc/pages/Signin.java]: delete just line 27 I think wait.until( ExpectedConditions.titleIs("Verify Your Identity | Salesforce"));

Comment (bruxita2312) [reply src/main/java/com/jalasoft/sdfc/pages/Signin.java]: Hi, 
Line 27, "Verify Your Identity | Salesforce", was removed from Signin.java file

File: src/test/resources/features/Signin.feature (hunk 2 above)
Comment (bruxita2312) [reply src/test/resources/features/Signin.feature]: Hi, 
User credentials were removed from Signin.feature file

**Review by acamacho82 — APPROVED** [review]

**Review by edgarquispe — APPROVED** [review]
the issues were fixed

**Review by mfuentes-ME — APPROVED** [review]
It is OK.

**Pull request merged**